app/controllers/users_controller.rb

```diff
@@ -56,9 +56,10 @@ def show
     username = params[:username].tr('_', ' ')
     @user = User.find_by_username(username)
     if @user
-      @courses_user = @user.courses_users
-      @courses_list = @user.courses.where('courses_users.role = 1')
-      @presenter = CoursesPresenter.new(current_user: current_user, courses_list: @courses_list)
+      @courses_users = @user.courses_users
+      @courses_list = @user.courses.where('courses_users.role = ?', CoursesUsers::Roles::INSTRUCTOR_ROLE)
+      @courses_presenter = CoursesPresenter.new(current_user: current_user, courses_list: @courses_list)
+      @users_presenter = UsersPresenter.new(user: @user)
```

Comment:
The name `UsersPresenter` doesn't do much clarify the specific role of this presenter. Maybe `IndividualContributionsPresenter`?